packages/next/server/incremental-cache.ts

```diff
@@ -80,6 +80,11 @@ export class IncrementalCache {
       )
     }
 
+    if (process.env.__NEXT_TEST_MAX_ISR_CACHE) {
+      // Allow cache size to be overridden for testing purposes
+      max = parseInt(process.env.__NEXT_TEST_MAX_ISR_CACHE, 10)
+    }
```

Comment:
Solution inspired by https://github.com/vercel/next.js/pull/24807#pullrequestreview-695168100

Reply:
@stefanb2's feature suggestion would also solve this nicely https://github.com/vercel/next.js/pull/27337. Those of us pushing the cache to its limit may want to configure this if we have memory to spare.